### DISCUSSION TITLE
Keyboard shortcut for exporting to H5 analysis files

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#1357] [Ideas]
Easy feature request that can save time when doing this repeatedly via the GUI.

Basically just a hotkey for:
https://github.com/talmolab/sleap/blob/1a0404c0ffae7b248eb360562b0bb95a42a287b6/sleap/gui/commands.py#L331

Menu item is added here:
https://github.com/talmolab/sleap/blob/1a0404c0ffae7b248eb360562b0bb95a42a287b6/sleap/gui/app.py#L474-L485

So we might just need to add `"export_analysis_current"` to the `Shortcuts` here:
https://github.com/talmolab/sleap/blob/1a0404c0ffae7b248eb360562b0bb95a42a287b6/sleap/gui/shortcuts.py#L10

And set a default in [`shortcuts.yaml`](https://github.com/talmolab/sleap/blob/develop/sleap/config/shortcuts.yaml).

Idea credit: Eartha Mae

**Comment by roomrys** [comment]
This idea has been merged into develop (see #1414) thanks to @KevinZ0217 and will be available in the next release SLEAP 1.3.2. The shortcut to export the analysis file for the current video is `Ctrl + E`.
